{
    "Computer":  "VSDC1",
    "Events":  [
                   {
                       "TimeCreated":  "\/Date(1761312266341)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start \"\\Microsoft\\Windows\\Workplace Join\\Automatic-Device-Join\" task for user \"(default user)\". Additional Data: Error Value: 2147943568.",
                       "Id":  101,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  202918
                   },
                   {
                       "TimeCreated":  "\/Date(1761312108834)\/",
                       "Source":  "Microsoft-Windows-DistributedCOM",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "DCOM got error \"1115\" attempting to start the service TrustedInstaller with arguments \"Unavailable\" in order to run the server:\r",
                       "Id":  10005,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  231877
                   },
                   {
                       "TimeCreated":  "\/Date(1761312092053)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to launch action \"Certificate Services Client Task Handler\" in instance \"{fa9870ff-c106-4c5c-bd99-aa0c170ec34b}\" of task \"\\Microsoft\\Windows\\CertificateServicesClient\\SystemTask\". Additional Data: Error Value: 2147943515.",
                       "Id":  203,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  202905
                   },
                   {
                       "TimeCreated":  "\/Date(1761312026320)\/",
                       "Source":  "Microsoft-Windows-Kerberos-Key-Distribution-Center",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "This event indicates an attempt was made to use smartcard logon, but the KDC is unable to use the PKINIT protocol because it is missing a suitable certificate.",
                       "Id":  19,
                       "ErrorCode":  null,
                       "UpdateTitle":  null,
                       "EventRecordID":  231871
                   },
                   {
                       "TimeCreated":  "\/Date(1761303663659)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to start instance \"{cc47c19c-7a62-4d17-86f9-2990e7474610}\" of \"\\Microsoft\\Windows\\Wininet\\CacheTask\"  task for user \"\" . Additional Data: Error Value: 2147942404.",
                       "Id":  103,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  202873
                   },
                   {
                       "TimeCreated":  "\/Date(1761303663659)\/",
                       "Source":  "Microsoft-Windows-TaskScheduler",
                       "LogName":  "Microsoft-Windows-TaskScheduler/Operational",
                       "Level":  "Error",
                       "Message":  "Task Scheduler failed to complete task \"\\Microsoft\\Windows\\Wininet\\CacheTask\" , instance \"{cc47c19c-7a62-4d17-86f9-2990e7474610}\" , action \"Wininet Cache task object\" . Additional Data: Error Value: 2147942404.",
                       "Id":  202,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  202872
                   },
                   {
                       "TimeCreated":  "\/Date(1761301050150)\/",
                       "Source":  "Microsoft-Windows-Perflib",
                       "LogName":  "Application",
                       "Level":  "Error",
                       "Message":  "Windows cannot load the extensible counter DLL \"C:\\Windows\\System32\\iasperf.dll\" (Win32 error code The specified module could not be found.).",
                       "Id":  1023,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  101652
                   },
                   {
                       "TimeCreated":  "\/Date(1761282637038)\/",
                       "Source":  "Volsnap",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The shadow copies of volume \\\\?\\Volume{992ed6e2-b097-11f0-ab89-00155d611102} were aborted during detection because a critical control file could not be opened.",
                       "Id":  27,
                       "ErrorCode":  "0x0001",
                       "UpdateTitle":  null,
                       "EventRecordID":  231396
                   },
                   {
                       "TimeCreated":  "\/Date(1761281604641)\/",
                       "Source":  "NETLOGON",
                       "LogName":  "System",
                       "Level":  "Error",
                       "Message":  "The session setup from the computer LTNC11525 failed to authenticate. The following error occurred: \r",
                       "Id":  5805,
                       "ErrorCode":  {

                                     },
                       "UpdateTitle":  null,
                       "EventRecordID":  231349
                   }
               ],
    "LastPoll":  "10/24 17:22"
}